### ISSUE TITLE
Access regression, classification and masks submodel outputs

### ISSUE THREAD

**Issue by tpostadjian** [fizyr/keras-maskrcnn#103]
Hi,

Can one easily retrieve outputs from the regression, classification and masks submodel ?
The following trick cannot work because of the submodel embedding in a keras.Models object for each of these processes (different graphs - Keras won't allow to mix their inputs/outputs):
```python
model = maskrcnn

regression_feature_extractor = keras.Model(inputs = model.input, outputs= model.get_layer(name="regression_submodel").get_layer("pyramid_classification_3").output)
  
classification_feature_extractor = keras.Model(inputs = model.input, outputs= model.get_layer(name="classification_submodel").get_layer("pyramid_classification_3").output)

mask_feature_extractor = keras.Model(inputs = model.input, outputs= model.get_layer(name="mask_submodel").get_layer("pyramid_classification_3").output)
```

Yet, I would like to get the features so as to perform further computations on these. 
One way would be to build one model for each task, but model slicing in Keras is not an easy task, especially for non-sequential models (with different streams as maskrcnn exhibits), where tracking inbounds nodes at each layer is not convenient when you have several inputs for some layers (P3 --> P7 feature maps).  

Any easy idea, workaround to get this done ?

Thanks !

**Comment by vcarpani** [comment]
Hi, the easiest way to do this would be to fork this repo and pass them as outputs in the model creation
